_src/static/sass/utilities/_variables.scss

```diff
@@ -15,7 +15,7 @@
 // ----------------------------
 // #UNITS
 // ----------------------------
-$em-base: 18; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
+$em-base: 16; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
```

Comment:
NOTE: changing ``em-base`` to match our base font-size in revised visual design.